docs/book/src/clusteropenstack/configuration.md

```diff
@@ -0,0 +1,309 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
```

Comment:
Should we already drop the duplicates in ./docs/* ?
(we can also do it after the book is published, I just don't want to have to keep them in sync long-term)